Makefile

```diff
@@ -17,22 +17,14 @@ LDFLAGS=$(shell ./scripts/version-ldflags)
 ############################### make help #####################################
 .PHONY: help
 help:  ## Display this help
```

Comment:
Could you confirm that `make help` works with this change? The last line is shown to be commented out.

Reply:
I did test it and it works for me.